components/schema-migrator/main.go

```diff
@@ -152,3 +220,50 @@ func (l *Logger) Printf(format string, v ...interface{}) {
 func (l *Logger) Verbose() bool {
 	return false
```

Comment:
?

Reply:
It's required by the migration library to implement the interface.